orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/pipeline/cluster/ShrinkClusterStage.groovy

```diff
@@ -49,8 +49,14 @@ class ShrinkClusterStage extends AbstractClusterWideClouddriverOperationStage {
         remainingEnabledServerGroups  : stage.context.shrinkToSize,
         preferLargerOverNewer         : stage.context.retainLargerOverNewer,
         continueIfClusterNotFound     : stage.context.shrinkToSize == 0,
-        interestingHealthProviderNames: stage.context.interestingHealthProviderNames
       ]
+
+      // We don't want the key present if interestingHealthProviderNames isn't defined.
+      // If 'Show health override option for each operation' is _disabled_, this key won't be overwritten if it already exists.
```

Comment:
Not sure this second line of the comment is really required. The logic exists elsewhere, and hasn't change as part of this pr.